intro.html

```diff
@@ -4,9 +4,11 @@
 
 <h2>What Can JavaScript Do?</h2>
 
-<p id="demo">My new proj.</p>
+<p id="try">My new proj.</p>
 
-<button type="button" onclick='document.getElementById("demo").innerHTML = "Hello Script!"'>Click here!</button>
+<button type="button" onclick='document.getElementById("try").innerHTML = "Hello Script!"'>Click here!</button>
+<p>hai how are you?<p/>
```

Comment:
please check and correct the tag closing

Reply:
changed the tag